openshift-container-platform-4/policies/SA-System_and_Services_Acquisition/component.yaml

```diff
@@ -136,10 +258,28 @@
 - control_key: SA-4 (6)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
-    - text: |
-        'A control response to SA-4 (6) is planned.'
+    - key: a
+      text: |
+        Organizational employment of only government off-the-shelf
+        (GOTS) or commercial off-the-shelf (COTS) information assurance
+        (IA) and IA-enabled information technology products that compose
+        an NSA-approved solution to protect classified information when
+        the networks used to transmit the information are at a lower
+        classificiation level than the information transmitted, is
+        an organizational control outside the scope of
+        OpenShift configuration.
+    - key: b
+      text: |
+        Organizational assurance that these products have been evaluated
+        and/or validated by NSA or in accordance with NSA-approved
+        procedures is an organizational control outside the scope
+        of OpenShift configuration.
+
+        NOTE: As of September 2019, OpenShift has not undergone formal NSA
```

Comment:
```suggestion
        NOTE: As of September 2019, OpenShift Container Platform 4.x has not undergone formal NSA
```

Reply:
Because OCP 3 on RHEL 7 had the "linux container framework" undergo Common Criteria testing, we likely should be specific about version number here.